README.md

```diff
@@ -67,7 +83,10 @@ Belle is friends with Mrs. Potts
 ```
 
 ```rb
-# Your code goes here...
+friends.map |friend| do
```

Comment:
:+1: but the question asked you to use `.each`. Your answer will still provide the desired effect, but we only use `.map` if we want to generate a new collection based off its code block.